### PR TITLE
Align naming in WiFi Network Diagnostics cluster XML with the spec.

### PR DESCRIPTION
**Pull request by bzbarsky-apple**
REVIEW NOTE: Only the first commit contains manual changes. The rest is code-generated.

### CODE REVIEW

**Review by andy31415 — APPROVED** [review]

**Review by tcarmelveilleux — APPROVED** [review]

File: src/darwin/Framework/CHIP/templates/availability.yaml [hunk 1]
```diff
@@ -5272,6 +5286,14 @@
...
               BatChargeLevel:
                   - Ok
+          WiFiNetworkDiagnostics:
+              WiFiVersionType:
+                  - 80211a
+                  - 80211b
+                  - 80211g
+                  - 80211n
+                  - 80211ac
+                  - 80211ax
```
Comment (ReneJosefsen) [inline src/darwin/Framework/CHIP/templates/availability.yaml:5296]: Stupid question but I'm wondering why 80211 is here when it is being removed everywhere else?

File: src/darwin/Framework/CHIP/templates/availability.yaml [hunk 2]
```diff
@@ -5354,6 +5379,14 @@
...
               BatChargeLevel:
                   OK: Ok
+          WiFiNetworkDiagnostics:
+              WiFiVersionType:
+                  A: 80211a
+                  B: 80211b
+                  G: 80211g
+                  N: 80211n
+                  Ac: 80211ac
+                  Ax: 80211ax
```
Comment (ReneJosefsen) [inline src/darwin/Framework/CHIP/templates/availability.yaml:5389]: Stupid question but I'm wondering why 80211 is here when it is being removed everywhere else?

**Review by ReneJosefsen — APPROVED** [review]

File: src/darwin/Framework/CHIP/templates/availability.yaml (hunk 1 above)
Comment (jtung-apple) [reply src/darwin/Framework/CHIP/templates/availability.yaml:5296]: If you look at line 5222, you'll see this is in the "deprecated" section, and so old values are put in here so auto-generated things can properly map old to new, for deprecation macros.

File: src/darwin/Framework/CHIP/templates/availability.yaml (hunk 2 above)
Comment (jtung-apple) [reply src/darwin/Framework/CHIP/templates/availability.yaml:5389]: Copy/pasting my comment from the other instance:

If you look at line 5222, you'll see this is in the "deprecated" section, and so old values are put in here so auto-generated things can properly map old to new, for deprecation macros.

**Review by jtung-apple — APPROVED** [review]

File: src/darwin/Framework/CHIP/templates/availability.yaml (hunk 1 above)
Comment (ReneJosefsen) [reply src/darwin/Framework/CHIP/templates/availability.yaml:5296]: Had a similar question in another PR and got an answer there as well, thanks for clarifying.

File: src/darwin/Framework/CHIP/templates/availability.yaml (hunk 2 above)
Comment (bzbarsky-apple) [reply src/darwin/Framework/CHIP/templates/availability.yaml:5389]: This is actually the "renames" section.  It's going here so the codegen knows that "A" used to be called "80211a" and can output the right old-and-new definitions and the right deprecation messages.

**Pull request merged**